mslice/plotting/pyplot.py

```diff
@@ -795,7 +801,8 @@ def subplot(*args, **kwargs):
           # first, the plot (and its axes) previously created, will be removed
           plt.subplot(211)
           plt.plot(range(12))
-          plt.subplot(212, facecolor='y') # creates 2nd subplot with yellow background
+          plt.subplot(212, facecolor='y') # creates 2nd subplot with yellow
+          background
```

Comment:
Is this a typo? (the `background` on the second line) - it doesn't give any errors when run but...

Reply:
I think it's okay. It's in a docstring so it's wrapped with a new line to indent it with the rest of the string.